python/lsst/ts/ATPneumaticsSimulator/pneumatics_csc.py

```diff
@@ -330,14 +330,14 @@ def set_cell_vents_events(self, closed, opened):
                 position=ATPneumatics.VentsPosition.OPENED,
             )
             self.evt_cellVentsState.set_put(
-                state=ATPneumatics.CellVentState.CELLVENTSOPENED,
+                state=ATPneumatics.CellVentState.CELLVENTSOPENEDSTATE,
```

Comment:
The XML has CellVentState_CellVentsOpened which means this should be spelled CELLVENTSOPENED.